datastore-api/tests/conftest.py

```diff
@@ -98,6 +99,48 @@ def test_node() -> Document:
         }
     )
 
+@pytest.fixture(scope="package")
+def test_node_lower() -> Document:
+    return Document(
+        __root__={
+            "id": "n1010",
+            "name": 'barack_obama',
+            'type': 'node',
+            'description': '44th president of USA',
+            'weight': None,
+            'in_id': None,
+            'out_id': None,
+        }
+    )
+
+@pytest.fixture(scope="package")
+def test_node_michelle() -> Document:
+    return Document(
+        __root__={
+            "id": "n1234",
+            "name": 'michelle_obama',
+            'type': 'node',
+            'description': 'First Lady of 44th president of USA',
+            'weight': None,
+            'in_id': None,
+            'out_id': None,
+        }
+    )
+
+@pytest.fixture(scope="package")
+def test_edge_married() -> Document:
+    return Document(
+        __root__={
+            "id": "e737",
+            "name": 'related_to',
+            'type': 'node',
+            'description': 'First Lady of 44th president of USA',
+            'weight': 1.0,
+            'in_id': "n1010",
+            'out_id': "n1234",
+        }
+    )
+
```

Comment:
These are unrelated to this PR. Please remove them and create a new PR if needed